sopel/loader.py

```diff
@@ -228,6 +247,26 @@ def is_url_callback(obj):
 
 
 def clean_module(module, config):
+    """Clean a module and return its callables. (commands, rules, jobs, etc.)
```

Comment:
Wild idea, but what if:

```suggestion
    """Clean a module and return its command, rule, job, etc. callables.
```

Reply:
SO WILD